includes/api/v1/class-wpsms-api-newsletter.php

```diff
@@ -175,7 +176,7 @@ public function unsubscribe_callback(WP_REST_Request $request)
     {
         // Get parameters from request
         $params         = $request->get_params();
-        $number         = Helper::convertNumber($params['mobile']);
+        $number         = NumberParser::toEnglishNumerals($params['mobile']);
```

Comment:
I think we also can use `getValidNumber` instead of `toEnglishNumerals` here as well.